lib/src/rules/spec/required.spec.ts

```diff
@@ -63,6 +63,20 @@ describe('[required] validation rule tests =>', () => {
       expect(validationResult.errorMessage).to.be.empty;
     });
 
+    it('should return true if typeof value is number', () => {
+      // Arrange
+      const value = 1;
```

Comment:
What is the validation if value is `0` or negative?

Reply:
Talking with @brauliodiez comments that we should not consider negative values, nor zero. That is, the validation fail in these cases:

- Value equals undefined
- Value equals null
- Value equals ' '

For another value, should pass validation:
- 'some text'
- 0
- 1
- -1
- {}
- []
- { property: 'value' }
etc.

@v-borrego, what do you think? Could you add some unit tests for these cases?